assets/package.json

```diff
@@ -30,6 +30,9 @@
     "webpack-cli": "^2.0.10"
   },
   "jest": {
-    "testRegex": "/test/.*_test\\.js$"
+    "testRegex": "/test/.*_test\\.js$",
+    "globals": {
+      "__PACKAGE_VERSION__": "test_mock_version"
```

Comment:
Shouldn't this be the same version as the package version? Or am I misunderstanding what "test_mock_version" is? :)

Reply:
Hi Jose, sorry for taking so long to respond. So, following the suggestion made by @ccblaisdell , the version on the JS side is injected via webpack when building and is exposed in the code as a global var. In this case the tests that directly call the js code don't have this treatment, so this is a way of exposing the global constant. This could be the manually set to the same one as the real version, but it could easily get out of date, since instead of testing if it is the same one it would only test that the var defined here is the same as in the result of the executed js function.